resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/ui/DriverPage.scala

```diff
@@ -153,6 +153,11 @@ private[ui] class DriverPage(parent: MesosClusterUI) extends WebUIPage("driver")
     <tr>
       <td>Memory</td><td>{driver.mem}</td>
     </tr>
+    <tr>
+      <td>Queue</td><td>{
+      driver.conf.get("spark.mesos.dispatcher.queue", config.DISPATCHER_QUEUE.defaultValueString)
+    }</td>
```

Comment:
Shall we follow the existing style at `def retryRow` like the following?
```scala
<td>Queue</td>
<td>
  {driver.conf.get("spark.mesos.dispatcher.queue", config.DISPATCHER_QUEUE.defaultValueString)}
</td>
```

Reply:
I formatted the Scala code following the [Code Style Guide](https://spark.apache.org/contributing.html) and by running `./dev/scalafmt`. I have complaints (formatting errors) for lines that are longer than 100 characters otherwise.

Reply:
The problematic point is `}</td>` part at line 159. You can split the inner line if it's long.

Reply:
I confirm: with your formatting I get "File line length exceeds 100 characters"

Reply:
Like this, https://github.com/apache/spark/pull/30352#discussion_r524479806

Reply:
Formatted :heavy_check_mark: